packages/frontend/core/src/components/pure/workspace-slider-bar/workspace-card/index.tsx

```diff
@@ -196,21 +190,47 @@ const useSyncEngineSyncProgress = () => {
       ) : (
         <LocalWorkspaceStatus />
       ),
+    active: syncing || retrying || isOverCapacity,
```

Comment:
Maybe we can add a debounce here

Reply:
I want the sync here not flicker, and the sync progress bar not lag (the old debounce would cause this).

Do you have any good ideas?

Reply:
https://github.com/toeverything/AFFiNE/blob/729f3f42da50f6597a5485c646db94a16365391c/packages/frontend/core/src/components/pure/workspace-slider-bar/workspace-card/styles.css.ts#L30

After adding this delay, animation won't play unless it takes more than 0.3s to sync.